examples/daisyworld.jl

```diff
@@ -335,3 +336,65 @@ p2 = plot(agent_df[!, :step], agent_df[!, aggname(adata[3])], ylabel = "temperat
 p3 = plot(model_df[!, :step], model_df[!, :solar_luminosity], ylabel = "L", xlabel = "ticks")
 
 plot(p, p2, p3, layout = (3, 1))
+
+
+# ## Interactive scientific research
+# Julia is an interactive language, and thus everything that you do with Agents.jl can be
+# considered interactive. However, we can do even better by using our interactive application.
+# In this example, we care about exploring some time-dependent forcing of the system, and
+# so far we have been pre-describing it. But with using the interactive application,
+# we can control by ourselves, in real time, how much to change the solar forcing.
+
+# So, let's use [`interactive_abm`](@ref)!
+
+# ```julia
+# using InteractiveChaos, Makie, Random
+# Random.seed!(165)
```

Comment:
Do we need the Random seed to be explicit here?

Reply:
Yes. Being explicitly is always better and I am not sure why the decision to hide `seed!` in the previous parts of the code was made.

Reply:
Simple enough to drop that in the docs then. We can make it consistent across the board.